python/tskit/trees.py

```diff
@@ -4687,6 +4705,13 @@ def draw_svg(
             be embedded in the root ``<svg>`` tag of the generated drawing.
         :param str style: A `css string <https://www.w3.org/TR/CSS21/syndata.htm>`_
             that will be included in the ``<style>`` tag of the generated svg.
+        :param str order: A string specifying the traversal type used to order the tips
+            in each tree, as detailed in :meth:`Tree.nodes`. If None (default), use
+            the default order as described in that method.
+        :param bool root_branch: Should we plot branches (edges) above the roots of each
+            tree? If None (default) if any root in the tree sequence has a mutation
+            above it, plot root branches for all roots in all trees, otherwise do not
+            plot root branches for any tree.
```

Comment:
Is this right? On line `565` the logic is `if self.mutations_over_root or root_branch`. Does that mean that the root branch always gets drawn if there are mutations over root?

Reply:
Good spot! Yes, the root branch is always drawn if there are mutations over the root. But perhaps you are right, and if `root_branch==False` we should not draw a branch, even if there are mutations that we should put there?

Reply:
I don't think it make sense to have a floating mutation, so the two options of "root branch drawn always"(`True`) and the default "root branch drawn if mutation above root" (`None`) seem fine.

Reply:
Although if you see a use case keep a `False` option in.

Reply:
Only use case I can see is a different one: plot a tree sequence, but only put a root branch for those trees that have a mutation above the root.

Reply:
Maybe that could be the default for tree sequence drawing?

Reply:
Keeping the options the same for tree-sequence and tree would be nice - so how about `None` does what you just said for tree and tree-sequence and `True` forces the root for both?

Reply:
I think that if you are plotting a tree sequence, you want the trees to look consistently the same (so you can compare them by eye) regardless of whether they have mutations above the root or not. That's also the case for a single "tree" with multiple roots: would it not be confusing if some nodes have a root branch and some not (see the SVG created in `test_no_edges()`)? So I think the current behaviour is correct: by default, if there is any need for a root branch, all trees should have it. You can also force there *always* to be a branch. There might be a case for an "explicitly only per tree" option too. Perhaps this is a parameter naming thing?

Reply:
> Perhaps this is a parameter naming thing?

So I think what we want is a better name for `force_roots_with_no_mutations_to_have_a_branch = False|None|True` where `None` (default) means "only if there roots elsewhere in the plot with a mutation above them". Hmm. Horrible names.

Reply:
I think you're right that the current behaviour is good, just so I'm right that's for tree-sequence and tree `force_root=True` always draws the root branch. For `None` tree draws root branch if mutation, and tree sequence draws root branches on all trees if one has a mutation.

That means you can't draw a tree sequence with only the trees that have mutations having the root branch. But I think that's ok, as you say better if they are consistent. Stick with `force_root`, can't think of anything better.

Reply:
So just to check - I don't think anything needs to be done here? Unless the docs need clarifying?

Reply:
Yeah, something like:
:param bool force_root_branch: If true plot a branch (edge) above each root of each tree. If None (default) then only plot such root branches if any root in the tree sequence has a mutation above it.
Is that right?

Reply:
Done in 2nd commit of latest rebase. Hope the wording is clear @benjeffery